wazo_webhookd/controller.py

```diff
@@ -50,6 +54,7 @@ def __init__(self, config):
                 'api': api,
                 'bus_consumer': self._bus_consumer,
                 'config': config,
+                'auth_client': self._auth_client,
```

Comment:
I have remove this.

It's currently passed to service plugins, too. I will do this change in an other PR

Reply:
Ah yeah I didn't see it. For http plugin, my main argument it's for homogeneity, but for service plugin, since only webhookd have it, I don't have any argument ... :sweat_smile: